christophermilian/climate-service
# DEV-4: Set up local development tooling

# Task
- [Set up local dependency tooling](#4)

# Description
- Set up the project dependencies
- Created an empty `index.ts` file to hold server information

# Testing
1. Run `nvm use`
2. Run `npm i` to install the local dependencies
3. Run `npm run local`
4. Verify that `nodemon` is waiting for changes
